github4s/src/main/scala/github4s/domain/Repository.scala

```diff
@@ -30,11 +30,13 @@ final case class RepositoryBase(
     updated_at: String,
     pushed_at: String,
     status: RepoStatus,
+    default_branch: String,
```

Comment:
_question_: Would it make sense to add the new fields at the end? Thinking about an easier way to upgrade for people using the existing model :)

Reply:
I like to keep the required fields before the fields having a default value (hence the `default_branch` here and the `topics` all the way down) but I can definitely change it if you think it would be better :-)

Reply:
Your point makes sense to me too, let's keep it that way 👍